bulkrax.gemspec

```diff
@@ -31,5 +31,4 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'sqlite3', '~> 1.3.13'
   s.add_development_dependency 'simplecov'
-  s.add_development_dependency 'bixby'
```

Comment:
Unsure if this is an okay change to make, but moving `bixby` to its own `group` in the Gemfile really sped up the `bundle install`.

Reply:
I'm good with it. Julie?

Reply:
me too